sys-process/nvtop/metadata.xml

```diff
@@ -30,5 +30,7 @@
 	</longdescription>
 	<use>
 		<flag name="unicode">Enable support for ncurses Unicode choice</flag>
+		<flag name="video_cards_panfrost">Enable support for Panfrost videocards</flag>
```

Comment:
There is no need to add use flag here, you just need to modify ``profiles/desc/video_cards.desc`` and add something like (you can modify) 

```
...
ascend - VIDEO_CARDS setting to build driver for Huawei ascend cards
...
panthor - VIDEO_CARDS setting to build driver for ARM Mali Gen10 GPU
...

```

`panfrost` already exists.
Note that this file must be sorted in lexicographic order